### DISCUSSION TITLE
Docker部署可以不下载二进制文件吗？

### DISCUSSION THREAD

**Discussion by neobuilding** [Harry-zklcdc/go-proxy-bingai#112] [Q&A] [answered]
@Harry-zklcdc  请问[Docker部署](https://github.com/Harry-zklcdc/go-proxy-bingai/wiki/Docker-%E9%83%A8%E7%BD%B2)必须下载二进制文件吗？可以直接本地编译吗？Dockerfile里那句go build就是用来编译的吧？直接跑一下docker build就行吧？

**Comment by Harry-zklcdc** [comment, marked as answer]
已经实现，看Wiki

**Reply by neobuilding** [reply]
@Harry-zklcdc 哦，看到了，现在改成直接下载你编译好的docker image了是吧？谢谢！
另外，wiki里可否提供一下本地编译的步骤呢？就是把本地的source code编译成Docker image
